src/components/shared/Entity/Details/Overview/DetailsSection/index.tsx

```diff
@@ -182,7 +187,7 @@ function DetailsSection({ entityName, latestLiveSpec }: DetailsSectionProps) {
             {!hasLength(data) ? (
                 <CircularProgress />
             ) : (
-                <KeyValueList data={data} />
+                <KeyValueList data={[...data, ...relatedEntities]} />
```

Comment:
Easier to just merge in the related entities array so we do not need to cause the memo run more than it needs to.